### PR TITLE
Prepare release 0.7.0

### PR DESCRIPTION
**Pull request by pracucci**
**What this PR does**:
In this PR I've prepared the release of Cortex `0.7.0`.

**Which issue(s) this PR fixes**:
N/A

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -165,11 +269,9 @@ Further, if you're using the configs service, we've upgraded the migration libra
 * [FEATURE] Added support to store ring tokens to a file and read it back on startup, instead of generating/fetching the tokens to/from the ring. This feature can be enabled with the flag `-ingester.tokens-file-path`. #1750
 * [FEATURE] Experimental TSDB: Added `/series` API endpoint support with TSDB blocks storage. #1830
 * [FEATURE] Experimental TSDB: Added TSDB blocks `compactor` component, which iterates over users blocks stored in the bucket and compact them according to the configured block ranges. #1942
-* [FEATURE] Experimental: Implements gRPC streaming for ingesters when using the experimental TSDB blocks storage. #1845
 * [ENHANCEMENT] metric `cortex_ingester_flush_reasons` gets a new `reason` value: `Spread`, when `-ingester.spread-flushes` option is enabled. #1978
 * [ENHANCEMENT] Added `password` and `enable_tls` options to redis cache configuration. Enables usage of Microsoft Azure Cache for Redis service. #1923
 * [ENHANCEMENT] Upgraded Kubernetes API version for deployments from `extensions/v1beta1` to `apps/v1`. #1941
-* [ENHANCEMENT] Add ability to configure gRPC keepalive settings. #2066
```
Comment (gouthamve) [inline CHANGELOG.md]: Weird that this entry made it here hrm,

Comment (pracucci) [reply CHANGELOG.md]: Was a very old PR and when we merged it was merged within the `0.6` release section (unfortunately due to a wrong rebase, which we didn't notice at review time).

File: CHANGELOG.md [hunk 2]
```diff
@@ -124,6 +128,106 @@ Cortex 0.4.0 is the last version that can *write* denormalised tokens. Cortex 0.
 
 Cortex 0.6.0 is the last version that can *read* denormalised tokens. Starting with Cortex 0.7.0 only normalised tokens are supported, and ingesters writing denormalised tokens to the ring (running Cortex 0.4.0 or earlier with `-ingester.normalise-tokens=false`) are ignored by distributors. Such ingesters should either switch to using normalised tokens, or be upgraded to Cortex 0.5.0 or later.
 
+### Known issues
+
+- The gRPC streaming for ingesters doesn't work when using the experimental TSDB blocks storage. Please do not enable `-querier.ingester-streaming` if you're using the TSDB blocks storage.
+
+### Annotated config file breaking changes
```
Comment (pstibrany) [inline CHANGELOG.md]: Very good idea!

File: CHANGELOG.md [hunk 3]
```diff
@@ -124,6 +128,106 @@ Cortex 0.4.0 is the last version that can *write* denormalised tokens. Cortex 0.
 
 Cortex 0.6.0 is the last version that can *read* denormalised tokens. Starting with Cortex 0.7.0 only normalised tokens are supported, and ingesters writing denormalised tokens to the ring (running Cortex 0.4.0 or earlier with `-ingester.normalise-tokens=false`) are ignored by distributors. Such ingesters should either switch to using normalised tokens, or be upgraded to Cortex 0.5.0 or later.
 
+### Known issues
+
+- The gRPC streaming for ingesters doesn't work when using the experimental TSDB blocks storage. Please do not enable `-querier.ingester-streaming` if you're using the TSDB blocks storage.
```
Comment (pstibrany) [inline CHANGELOG.md]: 👍 Perhaps we can mention that it should be fixed in master now, but being an experimental feature, it didn't make it to 0.7.0.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,11 +3,13 @@
 ## master / unreleased
 
 
-## 0.7.0-rc.0 / 2020-03-09
+## 0.7.0 / 2020-03-16
 
-Cortex `0.7.0` introduces some **breaking changes**. You're encouraged to read all the `[CHANGE]` entries below before upgrading your Cortex cluster. In particular:
+Cortex `0.7.0` is a major step forward the upcoming `1.0` release. In this release, we've got 164 contributions from 26 authors. Thanks to all contributors! ❤️
```
Comment (pstibrany) [inline CHANGELOG.md]: Pretty impressive!

File: CHANGELOG.md [hunk 5]
```diff
@@ -124,6 +128,106 @@ Cortex 0.4.0 is the last version that can *write* denormalised tokens. Cortex 0.
...
+-[configdb: <configdb_config>]
+
+ # "config_store" has been renamed to "configs".
+-[config_store: <configstore_config>]
++[configs: <configs_config>]
+
+
+### `distributor_config`
+
+ # The built-in billing system has been removed.
```
Comment (bboreham) [inline CHANGELOG.md]: I think I would describe this as a "hook to an external billing system"

**Pull request merged**